pkg/kubectl/resource_printer.go

```diff
@@ -84,15 +86,15 @@ func GetPrinter(format, formatArgument string) (ResourcePrinter, bool, error) {
 // ResourcePrinter is an interface that knows how to print runtime objects.
 type ResourcePrinter interface {
 	// Print receives a runtime object, formats it and prints it to a writer.
-	PrintObj(runtime.Object, io.Writer) error
+	PrintObj(runtime.Object, *watch.EventType, io.Writer) error
 }
 
 // ResourcePrinterFunc is a function that can print objects
-type ResourcePrinterFunc func(runtime.Object, io.Writer) error
+type ResourcePrinterFunc func(runtime.Object, *watch.EventType, io.Writer) error
```

Comment:
@smarterclayton  
Adding extra wrapper for HumanReadablePrinter will not work because there is already an extra wrapper on top of it - `VersionedPrinter`. 
The wrapper is applied here:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/util/factory.go#L397
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/get.go#L125